server/controllers/user-request.controllers.js

```diff
@@ -0,0 +1,77 @@
+import model from '../models';
+import serverError from '../helpers/server-error';
+
+const { User, Request } = model;
+
+const request = async (req, res) => {
+  const userId = req.user.userObj.id;
+  try {
+    const findUser = await User.findOne({
+      where: {
+        id: userId,
+        is_reviewer: false,
+        is_reported: false,
+      },
+    });
+    if (!findUser) {
+      return res.status(400).json({
+        status: 400,
+        errors: {
+          body: ['You are not allowed to make a request'],
+        },
+      });
+    }
+    const requested = await Request.create({
+      user_id: userId,
+      is_reviewer: false,
+      is_reported: false,
+    });
+    return res.status(201).json({
+      message: 'Successfully make a request',
```

Comment:
Should be `You have successfully made a request`

Reply:
Noted and fixed